hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OzoneManager.java

```diff
@@ -1706,21 +1706,24 @@ public boolean setOwner(String volume, String owner) throws IOException {
    * Changes the Quota on a volume.
    *
    * @param volume - Name of the volume.
-   * @param quota  - Quota in bytes.
+   * @param storagespceQuota - Quota in bytes.
+   * @param namespaceQuota - Quota in counts.
    * @throws IOException
    */
   @Override
-  public void setQuota(String volume, long quota) throws IOException {
-    if (isAclEnabled) {
+  public void setQuota(String volume, long namespaceQuota,
```

Comment:
Instead of `OzoneManager.java`, this request is processed in `OMVolumeSetQuotaRequest.java` .

We could not update code-snippet here.

Reply:
Similar to [setOwner](https://github.com/apache/hadoop-ozone/blob/7dac140024214c2189b72fad0566a0252d63e93c/hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OzoneManager.java#L1697), there already exists [setQuota](https://github.com/apache/hadoop-ozone/blob/7dac140024214c2189b72fad0566a0252d63e93c/hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OzoneManager.java#L1727). If none of this is needed, I think we can separate PR to clean up the useless methods.